express-post-json/server.js

```diff
@@ -0,0 +1,27 @@
+import express from 'express';
+const app = express();
+
+let nextId = 1;
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const gradesArray = [];
+  for (const key in grades) {
+    gradesArray.push(grades[key]);
+  }
+  res.json(gradesArray);
+});
+
+app.use(express.json());
```

Comment:
We usually put general purpose middleware like this at the top. This one isn't needed for the `app.get` so it can be here, but it doesn't hurt if it runs before `app.get` since if there is no JSON in the request, it does nothing. So it's safe to put at the top, and at the top is better organization of the code. We will be installing more general purpose middleware and having it all together makes things a bit more clear.